gym_connect_four/envs/connect_four_env.py

```diff
@@ -20,7 +20,24 @@ def __init__(self, env, name='Player'):
     def get_next_action(self, state: np.ndarray) -> int:
         pass
 
-    def learn(self, state, action, state_next, reward, done) -> None:
+    def learn(self, state, action: int, state_next, reward: int, done: int) -> None:
+        pass
+
+    def save_model(self, model_prefix: str = None):
```

Comment:
If you don't implement this method, mark it as abstract - so forcing being implemented, or throw an NotImplemented exception in case it's gonna be called in this way. Otherwise it would confuse that model was saved successfully.